object_pool.go

```diff
@@ -0,0 +1,45 @@
+package astits
+
+import "sync"
+
+// poolOfTempPayload global variable is used to ease access to pool from any place of the code
+var poolOfTempPayload = &poolTempPayload{
+	sp: sync.Pool{
+		New: func() interface{} {
+			// Prepare the slice of somewhat sensible initial size to minimize calls to runtime.growslice
+			return &tempPayload{
+				s: make([]byte, 0, 1024),
+			}
+		},
+	},
+}
+
+// tempPayload is an object containing payload slice
+type tempPayload struct {
+	s []byte
+}
+
+// poolTempPayload is a pool for temporary payload in parseData()
+// Don't use it anywhere else to avoid pool pollution
+type poolTempPayload struct {
```

Comment:
could you rename this struct to `bytesPooler`?

Reply:
Done